src/uwtools/config/validator.py

```diff
@@ -54,8 +55,8 @@ def validate_yaml(
     log_msg = "%s UW schema-validation error%s found"
     log_method(log_msg, len(errors), "" if len(errors) == 1 else "s")
     for error in errors:
-        for line in str(error).split("\n"):
-            log.error(line)
+        log.error("Error at %s:", " -> ".join(str(k) for k in error.path))
+        log.error("%s%s", INDENT, error.message)
```

Comment:
Cherry-pick the most useful information from each JSON Schema `ValidatonError` object.